CIP-0001/README.md

```diff
@@ -218,7 +218,7 @@ A _'Proposed'_ CIP is any CIP that meets the essential CIP criteria but is not y
 An _'Active'_ CIP has taken effect according to the criteria defined in its _'Path to Active'_ section. Said differently, each CIP defines by which criteria it can become _'Active'_ and those criteria are included in the review process. Exact criteria thereby depends on the nature of the CIP, typically:
 
 - For CIPs that relate to particular projects or pieces of technology, it becomes _'Active'_ by being implemented and released;
-- For changes to the Cardano protocol, a CIP becomes _'Active'_ by being live on the Cardano mainnet;
+- For changes to the Cardano protocol, a CIP becomes _'Active'_ by being live on the Cardano mainnet within one or more implementation;
```

Comment:
```suggestion
- For changes to the Cardano protocol, a CIP becomes _'Active'_ by being live on the Cardano mainnet within one or more implementations;
```
[grammar nitpick] "one or the other" would be singular, as it's singular in every case: but "one or more" has to accommodate the plural case: therefore this has to be "implementations". [[reference](https://english.stackexchange.com/questions/13284/which-is-correct-one-or-more-is-or-one-or-more-are)]